make/test.mk

```diff
@@ -0,0 +1,44 @@
+KOMMANDER_E2E_DIR  = $(REPO_ROOT)/.tmp/kommander-e2e
+
+# E2E configurations
+E2E_TIMEOUT       ?= 120m
+E2E_KINDEST_IMAGE ?= "kindest/node:v1.23.5"
+UPGRADE_FROM_VERSION ?= "release-2.2"
```

Comment:
we'll need to update some release automation tickets to make sure this gets updated to the previous release in each branch